src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+  const FEW_DAYS = 3;
+  const FEW_DAYS_DISCOUNT = 20;
+  const MANY_DAYS = 7;
+  const MANY_DAYS_DISCOUNT = 50;
+
+  const PRICE_PER_DAYS = COST_PER_DAY * days;
```

Comment:
```suggestion
  const currentPrice = COST_PER_DAY * days;
```
Variables should be in camelCase, as this dynamic variable that depends on the `days` it's not a constant